### PR TITLE
Add LD2410 BLE integration

### PR DESCRIPTION
**Pull request by 930913**
## Proposed change

Add a markdown page for the LD2410 BLE integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/83883
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3925

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ld2410_ble.markdown [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Integrates LD2410B sensors from [Hi-Link](http://www.hlktech.net/) into Home Assistant using Bluetooth.
+
+{% include integrations/config_flow.md %}
+
+It currently provides the motion and static value. Other values, including configuring it are TODO.
```
Comment (frenck) [inline source/_integrations/ld2410_ble.markdown]: We don't mention todo's in our documentation. Please remove that.

File: source/_integrations/ld2410_ble.markdown [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+You can currently configure the settings for motion/static triggers by USB with either the [desktop software](https://drive.google.com/drive/folders/1p4dhbEJA3YubyIjIIC7wwVsSo8x29Fq-?usp=sharing) or the Bluetooth [mobile app](https://www.pgyer.com/Lq8p).
```
Comment (frenck) [inline source/_integrations/ld2410_ble.markdown:25]: Why is this linking to a Google drive?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/ld2410_ble.markdown (hunk 1 above)
Comment (930913) [reply source/_integrations/ld2410_ble.markdown]: Removed.

File: source/_integrations/ld2410_ble.markdown (hunk 2 above)
Comment (930913) [reply source/_integrations/ld2410_ble.markdown:25]: For some arcane reason, the manufacturer has seen fit to distribute the software via a google drive share. This is the link they have provided elsewhere, and I have copied it here to ease the user's journey until the configuration can be done inside Home Assistant.

Comment (frenck) [reply source/_integrations/ld2410_ble.markdown:25]: Can we point to the manufacturer's website or documentation that provides that information and links? It seems very inefficient to use the drive link on third-party websites. They can easily expire/won't update. I rather avoid them for ending up in our documentation.

Comment (930913) [reply source/_integrations/ld2410_ble.markdown:25]: I'm not sure how. If you look at [their listing page on aliexpress](https://www.aliexpress.com/item/1005004351593073.html), [their comments on YouTube](https://www.youtube.com/watch?v=dAzHXpP3FcI&lc=UgyG9sIEjRotx485AgB4AaABAg.9h-Wb-MW1Br9itOezFW363), or their correspondence, that's the official link as far as they provide.
Short of uploading the contents into this repository and redistributing, I think we're stuck with their official link, as unofficial as it seems to us.

Comment (930913) [reply source/_integrations/ld2410_ble.markdown:25]: Bump?

Comment (frenck) [reply source/_integrations/ld2410_ble.markdown:25]: Thanks for clarifying. Nasty, but it is what it is I guess 🤷 

../Frenck

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by frenck — DISMISSED** [review]

File: source/_integrations/ld2410_ble.markdown [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ha_codeowners:
+  - '@930913'
+ha_domain: ld2410_ble
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Integrates LD2410B sensors from [Hi-Link](http://www.hlktech.net/) into Home Assistant using Bluetooth.
```
Comment (bdraco) [inline source/_integrations/ld2410_ble.markdown:19]: Is there an example of where can I purchase this device?

Comment (930913) [reply source/_integrations/ld2410_ble.markdown:19]: The main listing is https://www.aliexpress.com/item/1005004351593073.html, where there are a number of options - Bluetooth/non-Bluetooth, devkit/bare-board, pins/no pins. Obviously the non-Bluetooth variant won't work with this integration :wink:

Comment (bdraco) [reply source/_integrations/ld2410_ble.markdown:19]: Maybe warning that there are multiple modules of this module and users should be sure to purchase the -B version as the others don't have bluetooth.

Although that listing implies the C does as well but it is aliexpress so....

Comment (930913) [reply source/_integrations/ld2410_ble.markdown:19]: Purchase section added.

**Review by bdraco — APPROVED** [review]

**Pull request merged**